experimental/dds/tree/api-extractor/api-extractor.legacy.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
It doesn't look like this config file is consumed, otherwise I would expect to see new reports. If we want to properly differentiate the legacy and non-legacy exports in the reports, then we need to make more changes.

This PR is an example of a similar change, though client-utils is a bit more complex due to its separate `browser` and `node` exports: https://github.com/microsoft/FluidFramework/pull/25283/files

Reply:
Seems like it's not needed. Close this PR.